From f53cbbe1706827fe3a3888f64a4c4ce89868e14a Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Mon, 21 Sep 2015 11:21:29 +0800
Subject: [PATCH 0453/5242] MLK-11593-3 ARM: dts: imx6ul.dtsi: remove
 redundancy touch screen

commit  5d3324b091f2946ae5ebe7c0cf3e707819c65293 from
https://source.codeaurora.org/external/imx/linux-imx.git

Currently we only use 'tsc' for touch screen, so here remove
the redundancy 'touchctrl'.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul.dtsi |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index a9dce60..a2bc206 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -442,12 +442,6 @@
 				status = "disabled";
 			};
 
-			touchctrl: touchctrl@2040000 {
-				compatible = "fsl,imx6ul-touchctrl";
-				reg = <0x02040000 0x4000>;
-				status = "disabled";
-			};
-
 			bee: bee@2044000 {
 				compatible = "fsl,imx6ul-bee";
 				reg = <0x02044000 0x4000>;
-- 
1.7.9.5

